spec/predicates/human-review/crev.md

```diff
@@ -0,0 +1,127 @@
+# Predicate type: Dependency Reviews (crev)
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/crev/v0.1
+
+Version: Do we use Crev's versioning? Currently -1? What about their URI?
+
+## Purpose
+
+This attestation type is used to describe the results of human review of
+dependency source code. The format is based on the
+[crev project](https://github.com/crev-dev/crev).
+
+## Use Cases
+
+As noted above, crev enables social review of popular open source software
+dependencies. A crev review includes information such as the thoroughness of
+the review, understanding of the source code, and a final rating.
+
+## Prerequisite
+
+Understanding of the
+[in-toto attestation specification](https://github.com/in-toto/attestation).
+
+## Model
+
+Most modern software have external dependencies. Dependency review is the
+process of reviewing and verifying the source code of a particular
+dependency, and can be performed by one or more of several actors in the supply
+chain. The developer importing a new dependency can perform the review or a
+dedicated security team can be tasked with it.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [{...}],
```

Comment:
I think it would be helpful to have some language about what the subjects of these things are expected to be.

It looks like for crev it's expected to be a binary and not source code itself?

I also think it's fine to delegate the details to the various projects (crev doesn't seem to define this yet so maybe it's enough to just explicitly leave it open-ended)?

Reply:
> It looks like for crev it's expected to be a binary and not source code itself?

I understand it as a version of a package. I think resource descriptors and/or purl might fit the criteria. I'll update that.